styles/styles.css

```diff
@@ -0,0 +1,93 @@
+
+*{
+
+  padding: 0px;
+  border: 0;
+  margin: 0;
+  height: auto;
+  font-family: 'Roboto', sans-serif;
+
+}
+body
```

Comment:
It is a good idea to have an empty line between definitions, so in this case, before the "body". Make this changes everywhere in the file